src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatterTest.java

```diff
@@ -159,4 +155,8 @@ public void testPreposition() {
     public void testAvoidNameAffixes() {
         expectCorrect("Canon der Barbar, Alexander der Große","der Barbar, Canon and der Große, Alexander");
     }
+
+    private void expectCorrect(String input, String expected) {
```

Comment:
Could you remove this method to make the tests being consistent with other tests?

Please be aware that the order of parameters has to be swapped!